tendrl/commons/objects/cluster/atoms/configure_monitoring/__init__.py

```diff
@@ -0,0 +1,114 @@
+from tendrl.commons import objects
+from tendrl.commons.objects import AtomExecutionFailedError
+from tendrl.commons.utils.cmd_utils import Command
+from tendrl.commons.utils import log_utils as logger
+
+
+NODE_PLUGINS = {
+    'collectd',
+    'cpu',
+    'mount_point',
+    'memory',
+    'swap',
+    'network',
+    'latency',
+    'disk',
+    'dbpush'
+}
+
+
+class ConfigureMonitoring(objects.BaseAtom):
+    def __init__(self, *args, **kwargs):
+        super(ConfigureMonitoring, self).__init__(*args, **kwargs)
+
+    def _configure_plugin(self, plugin_name, plugin_params):
+        logger.log(
+            "info",
+            NS.get("publisher_id", None),
+            {
+                "message": "Starting configuration of %s on %s with %s"
+                "as conf parameters" % (
+                    plugin_name,
+                    NS.node_context.fqdn,
+                    plugin_params
+                )
+            },
+            job_id=self.parameters['job_id'],
+            flow_id=self.parameters['flow_id']
+        )
+        cmd_str = "config_manager %s '%s'" % (
+            plugin_name,
+            plugin_params
+        )
+        out, err, rc = Command(cmd_str).run()
```

Comment:
we should rename the "config_manager" command to "tendrl_monitoring_config_manager" to avoid clashing with other binarieis and white list that command here https://github.com/Tendrl/commons/blob/master/tendrl/commons/utils/cmd_utils.py#L18